config/environments/development.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Verifies that versions and hashed value of the package contents in the project's package.json
```

Comment:
What's this comment trying to say? 🤔

Reply:
Not sure, comment came with the upgrade..